draft-ietf-tls-exported-authenticator.md

```diff
@@ -270,6 +270,19 @@ A given authenticator can be validated by checking the validity of the
 CertificateVerify message given the authenticator request (if used) and recomputing the
 Finished message to see if it matches.
 
+# Empty Authenticator
+
+If, given an authenticator request, the endpoint does have an appropriate certificate
+or does not want to return one, it may construct an authenticated denial of existence
+called an empty authenticator.  This is an HMAC over the hashed authenticator
+transcript with the Certificate and CertificateVerify messages omitted:
```

Comment:
This isn't what TLS does.  It sends the Certificate, but with no certificate in it.

That's probably necessary here too because we use the `certificate_request_context`.

Reply:
The `certificate_request_context` is included in the `CertificateRequest`, and thus the `Finished` message is dependent on it. 
The initiator would have to guess which request is being denied, but the denial would still be dependent on the `certificate_request_context`.

Reply:
That's right.  This isn't a question of whether the context is included, but just simple correlation.  In the use of this for HTTP, we rely on the context.